.github/workflows/translators-sync.yml

```diff
@@ -0,0 +1,46 @@
+name: Sync Translators Branch
+
+permissions:
+  actions: write
+  checks: write
+  contents: write
+  deployments: write
+  pull-requests: write
+  statuses: write
+  
+on:
+  push:
+    branches:
+        - 'master*'
+        
+jobs:
+    sync-translation:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout 🛎️
+              uses: actions/checkout@v4.1.1
+            - name: Setup Node
+              uses: actions/setup-node@v4.0.0
+              with:
+                  node-version: '18.x'
+            - name: Sync Master to Translators branch 🔀
+              run: |
+                  branch_name="master_translations"
+
+                  echo "Setting up Git identity"
+                  git config --global user.name "DerivFE"
+                  git config --global user.email "80095553+DerivFE@users.noreply.github.com"
+
+                  echo "Checking out new branch [$branch_name]"
+                  git checkout -b "$branch_name"
+
+                  # Force push to this branch in case a previous run created it.
+                  git push --set-upstream origin "$branch_name" -f
+
+                  sudo apt install gh
```

Comment:
Can we add a check to not to install gh, if it's already installed?

Reply:
@habib-deriv does it take a long time, even if it's already installed? For me (in docker) the first install of `gh` takes 3.2s, but then second call to install is 0.5s:

```
root@2a81a4fdf583:/# time apt install gh
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
gh is already the newest version (2.4.0+dfsg1-2).
0 upgraded, 0 newly installed, 0 to remove and 7 not upgraded.

real	0m0.557s
user	0m0.504s
sys	0m0.054s
```